.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+# Related Issue
+- One line for each Issue. If merging this Pull Request should auto-close the Issue on that line,
```

Comment:
A few capitalization updates here:

There's generally no need to capitalize a word mid-sentence unless it's a name, such the name of person or product (like OpenCue). In this case, I think you can just use lower case for all of the following:

- pull request
- issue
- number

The capitalization of "Fixes" seems OK as it's an instruction for how a user should write something.

Reply:
Thanks for sharing such an important information.